.github/workflows/nightly.yml

```diff
@@ -0,0 +1,172 @@
+name: Nightly
+
+on:
+  push:
+  schedule:
+    - cron: 0 0 * * *
+
+jobs:
+
+  master:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: master
+
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
I believe we should use Node 18 now, correct? See https://github.com/privacy-scaling-explorations/maci/issues/775

Reply:
Yes for unit tests. But e2e test has problem with Node 18. See https://github.com/privacy-scaling-explorations/maci/pull/775#discussion_r1374742715